Libs/MRML/Widgets/qMRMLPlotView_p.h

```diff
@@ -81,21 +89,23 @@ public slots:
   void onPlotChartNodeChanged();
 
   void RecalculateBounds();
-  void switchSelectionMode();
+  void switchInteractionMode();
   void switchLeftAndMiddleClick();
```

Comment:
I guess this method can be removed now that the pan is done with left click.

Reply:
Yes. The mechanism was also a bit fragile (releasing shift when mouse was clicked outside the view left the view in inconsistent state).

Reply:
ok, perfect, then I please remove it